### ISSUE TITLE
Training settings

### ISSUE THREAD

**Issue by dwangxxx** [ericsujw/InstColorization#18]
Hello, I've read your paper recently. In this paper, you mentioned that the training epoch is set as 2, 5 and 2, but the source code is set 150. I want to know which is correct? Thank you!

**Comment by dwangxxx** [comment]
Hello, sorry to bother again. In this paper, you show the results finetuned on the COCOstuff dataset. I want to know how to finetune the model on the COCOstuff dataset for ImageNet dataset. I don't know if I'm right, but I thought of the following steps:
(1) Train the model on the ImageNet dataset.
(2) Finetune the model from (1) on the COCOStuff dataset with a smaller learning rate.
Maybe I was wrong, but can you provide me some detailed information, such as finetune learning rate and some other parameters?
